WalletWasabi.Gui/Styles/Styles.xaml

```diff
@@ -216,4 +216,20 @@
   <Style Selector="dock|ProportionalStackPanelSplitter">
     <Setter Property="IsHitTestVisible" Value="False" />
   </Style>
+
+  <Style Selector="shell|MetroWindow.ToolWindow /template/ Button#minimiseButton">
```

Comment:
might need to change these when we use WasabiWindow.

Reply:
@jmacato did you take a look at this?

Reply:
@molnard it shouldnt be changed to wasabiwindow since metrowindow owns the template instead of it.

Reply:
ACK!